main.js

```diff
@@ -220,6 +227,28 @@ define(function (require, exports, module) {
         }
     }
     
+    function _applyColors() {
+        // Slight delay so Brackets can apply the dark class
+        setTimeout(function() {
+            var userColors = _preferences.get("colors"),
+                toApply    = $("body").hasClass("dark") ? userColors.dark : userColors.light,
+                css        = ".CodeMirror .cm-dk-whitespace-space:before," +
```

Comment:
Oh duh. Sorry, it's late at night, I've had a long day, I shouldn't be coding in this state, and new stuff here so I'm having to learn and catch on. Sorry to ask such dumb questions at times.